glide2x/cvg/glide/src/Makefile.mingw

```diff
@@ -57,7 +57,7 @@ GLIDE_DLL = glide2x.dll
 GLIDE_DEF = glide2x.def
 GLIDE_IMP = libglide2x.dll.a
 
-FX_GLIDE_SW = ../../../swlibs
+FX_GLIDE_SW = ../../../../swlibs
```

Comment:
The actual intent is that `swlibs` is copied (or symlinked) under glide2x and glide3x directories and build after. So let's keep those `FX_GLIDE_SW` as they are.

Reply:
That is a little bit odd. What would be the purpose of copying those? Whatever the reason, I believe that if they should be copied, that should be part of the build process, or at the very least be documented. As it is right now, the build just fails and one wonders why.

Reply:
I guess the purpose is to create source tarballs with swlibs included. Yes I know that it's not intuitive when building from a git clone, but I don't want to change it for now. The djgpp cross-build scripts for e.g. does symlinking automatically for example.  Adding a note to a README would be useful, instead of changing.

Reply:
>  Adding a note to a README would be useful, instead of changing.

The glide2x README already seems to have a note about it:
https://github.com/sezero/glide/blob/glide-devel-sezero/glide2x/README#L27-L31

Will add a similar note to the glide3x README, shortly.